assets/libraries/openapi/library.rego

```diff
@@ -161,3 +161,11 @@ require_objects := {
 	"clientCredentials": {"value": {"scopes"}, "array": false, "map_object": false},
 	"authorizationCode": {"value": {"scopes"}, "array": false, "map_object": false},
 }
+
+api_key_exposed(doc, version, s) {
+	version == "3.0"
+	doc.components.securitySchemes[s].type == "apiKey"
+} else {
+	version == "2.0"
+	doc.securityDefinitions[s].type == "apiKey"
+}
```

Comment:
It is really necessary to move this check to library?

Reply:
It is the same function used in [here](https://github.com/Checkmarx/kics/pull/3596)